buildSrc/build.gradle.kts

```diff
@@ -9,3 +11,14 @@ repositories {
 kotlinDslPluginOptions {
     experimentalWarning.set(false)
 }
+
+val props = Properties().apply {
```

Comment:
please rename it to just `properties`

Reply:
`properties` already exists in `Project`